content/en/docs/tasks/debug/debug-cluster/troubleshoot-kubectl.md

```diff
@@ -123,23 +123,23 @@ directory. The `certificate-authority` attribute contains the CA certificate and
 Verify the expiry of these certificates:
 
 ```shell
-openssl x509 -noout -dates -in $(kubectl config view --minify --output 'jsonpath={.clusters[0].cluster.certificate-authority}')
+kubectl config view --flatten --output 'jsonpath={.clusters[0].cluster.certificate-authority-data}' | base64 -d | openssl x509 -noout -dates
```

Comment:
(nit)
I'd mention `base64` in the page prerequisites. Also `openssl`, although it should have been there anyway.

Reply:
It's not clear to me how to best advise base64 and openssl as pre-requisites. base64 is part GNU core utilities and for openssl, as you suggest, should be already there...from my perspective, if you are this far in i.e. troubleshooting, both of these requirements would be implicitly met.

Reply:
Hi - this is my first PR - is there something else that needs to be done for it to be merged? thankls.

Reply:
> It's not clear to me how to best advise base64 and openssl as pre-requisites. base64 is part GNU core utilities and for openssl, as you suggest, should be already there...from my perspective, if you are this far in i.e. troubleshooting, both of these requirements would be implicitly met.

Mention these in the [prerequisites](https://kubernetes.io/docs/tasks/debug/debug-cluster/troubleshoot-kubectl/#before-you-begin) section.
This is a little like a recipe; a recipe for a meal might say something like:

> - preheat the oven to 220°C
> - you will need a steamer basket and an ovenproof metal bowl at least 15cm deep
> 
> and a list of ingredients


Even though cooks reading that recipe probably have an oven and the utensils, listing them all is helpful. If someone wanted to do troubleshooting and they don't have `openssl`, it's helpful for them to know that right from the start.

Reply:
Hi - thanks for the suggestion. I considered putting into the [prerequisites](https://kubernetes.io/docs/tasks/debug/debug-cluster/troubleshoot-kubectl/#before-you-begin) but given not all issues in this document required those tools, I thought some additional tooling in the TLS problems passage would be cleaner - however, this may not comply with the doc standard so I am also ok to move it into the [prerequisites](https://kubernetes.io/docs/tasks/debug/debug-cluster/troubleshoot-kubectl/#before-you-begin) - just let us know.